src/main/java/uk/gov/hmcts/juror/api/moj/report/standard/PersonAttendingSummaryReport.java

```diff
@@ -0,0 +1,97 @@
+
+    @Autowired
+    public PersonAttendingSummaryReport(PoolRequestRepository poolRequestRepository,
+                                        CourtLocationRepository courtLocationRepository) {
+        super(poolRequestRepository,
+            QJurorPool.jurorPool,
+            DataType.JUROR_NUMBER,
+            DataType.FIRST_NAME,
+            DataType.LAST_NAME);
+
+        this.courtLocationRepository = courtLocationRepository;
+        isCourtUserOnly();
+        addAuthenticationConsumer(request -> checkOwnership(request.getLocCode(), false));
+    }
+
+    @Override
+    protected void preProcessQuery(JPAQuery<Tuple> query, StandardReportRequest request) {
+        query.where(QJurorPool.jurorPool.nextDate.eq(request.getDate()));
+        query.where(QJurorPool.jurorPool.pool.courtLocation.locCode.eq(request.getLocCode()));
+        if (request.getIncludeSummoned()) {
+            query.where(QJurorPool.jurorPool.status.status.in(IJurorStatus.SUMMONED,
+                                                              IJurorStatus.RESPONDED,
+                                                              IJurorStatus.PANEL,
+                                                              IJurorStatus.JUROR));
+        } else {
+            query.where(QJurorPool.jurorPool.status.status.in(IJurorStatus.RESPONDED,
+                                                              IJurorStatus.PANEL,
+                                                              IJurorStatus.JUROR));
+        }
+    }
+
+    @Override
+    public Map<String, StandardReportResponse.DataTypeValue> getHeadings(StandardReportRequest request,
+                                                                         StandardReportResponse.TableData<
+                                                                             List<LinkedHashMap<String, Object>>>
+                                                                             tableData) {
+
+        Map<String, StandardReportResponse.DataTypeValue> map = new ConcurrentHashMap<>();
+        map.put("total_due", StandardReportResponse.DataTypeValue.builder()
+            .displayName("Total due to attend")
+            .dataType(Integer.class.getSimpleName())
+            .value(tableData.getData().size())
+            .build());
+
+        map.put("attendance_date", AbstractReportResponse.DataTypeValue.builder().displayName("Attendance date")
+            .dataType(LocalDate.class.getSimpleName())
+            .value(DateTimeFormatter.ISO_DATE.format(request.getDate())).build());
+
+        map.put("court_name", AbstractReportResponse.DataTypeValue.builder().displayName("Court Name")
+            .dataType(String.class.getSimpleName())
+            .value(getCourtNameString(courtLocationRepository, request.getLocCode()))
+            .build());
+
+        return map;
+    }
+
+    @Override
+    public Class<? extends Validators.AbstractRequestValidator> getRequestValidatorClass() {
```

Comment:
? extends Validators.AbstractRequestValidator should be PersonAttendingSummaryReport.RequestValidator